content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+
+See the "API Streamer documentation":/api-streamer for further information. 
+
+h2(#apps). Apps
+
+An app (or application) in the context of Ably is a grouping of channels and other resources that are managed as a collection. Each app has its own set of API keys and can have its own rules and queues. An app defines a "sandbox" or namespace of channel and queue names, so they are are unique within an app and access by a client is generally confined to a single app. An app is associated with a unique account.
+
+h2(#batch-publishing). Batch publishing
+
+Batch publishing is the process of making multiple publish requests as part of a single request. For example, a message can be published to multiple channels using a single request.
+
+See the "batch publishing documentation":/rest-api/beta#batch for further information.
+
+h2(#capability). Capability
+
+A capability is an expression that defines a collection of resources. Each resource has a set of permitted operations, such as permission to subscribe to the @finance:@ channel namespace. Capabilities are used to define the rights associated with an authentication credential such as an API key or token, although the rights expressed by a capability for a token cannot exceed those of its associated API key. 
+
+See the "capabilities documentation":/core-features/authentication/#capabilities-explained for further information
+
+h2(#channels). Channels
+
+Channels are used to logically separate messages into different topics. Clients subscribe to channels to receive the messages sent on them. When a publisher sends a message on a channel, all subscribers receive that message.
+
+See the "channels documentation":/core-features/channels for further information
+
+h2(#channel-namespaces). Channel namespaces
+
+Channel namespaces provide a way for certain properties (such as persistence) to be assigned to a group of channels, rather than requiring them to be assigned to individual channels. The namespace is the first colon-delimited segment of a channel name, or the channel name itself if no colon exists. For example, the channels @finance@ and @finance:approvals@ would both be considered part of a @finance@ namespace.
+
+See the "channel namespace documentation":/realtime/channels#channel-namespaces for further information.
+
+h2(#connection). Connection
+
+A connection to Ably is must be established and maintained for realtime communication. A connection has several connection states throughout the lifecycle of a connection, such as @connecting@, @connected@ and @suspended@.
+
+See the "connection documentation":/realtime/connection for further information.
+
+h2(#dashboard). Dashboard
+
+The Ably dashboard is the user interface for managing applications, viewing statistics, creating API keys and setting up integrations and channel rules.
+
+Access your Ably dashboard by "logging in":https://ably.com/login or "signing up":https://ably.com/sign-up.     
```

Comment:
Can we say "Once you have "registered":https://ably.com/sign-up for an Ably account, you can "log in":https://ably.com/login to view your dashboard." ?